src/vs/workbench/browser/parts/editor/tabsTitleControl.ts

```diff
@@ -1174,6 +1188,10 @@ export class TabsTitleControl extends TitleControl {
 			tabContainer.style.backgroundColor = this.getColor(isGroupActive ? TAB_INACTIVE_BACKGROUND : TAB_UNFOCUSED_INACTIVE_BACKGROUND) || '';
 			tabContainer.style.boxShadow = '';
 
+			// bottom border when wrapping multi-line
```

Comment:
My suggestion was to reuse the `tab-border-bottom` helper for this case too and not introduce yet another border. Currently the `tab-border-bottom` is reserved for active tabs because the border was only used for active tabs, but to ensure we get a consistent look, we should be using the same thing for the separator as well. Otherwise we end up with an active border that is not on the exact same line as the separator border.

Reply:
Done, I misunderstood what you suggested earlier but hopefully it's correct now?